lib/rubocop/cop/internal_affairs/plugin.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'lint_roller'
+
+module RuboCop
+  module InternalAffairs
+    # A Plugin for `InternalAffairs` department, which has internal cops.
+    class Plugin < LintRoller::Plugin
+      def about
+        LintRoller::About.new(
+          name: 'rubocop/cop/internal_affairs',
```

Comment:
Does the name have to relate to the path? Maybe calling this `rubocop-internal-affairs` would be cleaner, as it would be loaded via the `plugins` configuration in other plugins, for instance.

Reply:
Yeah, this was a point I struggled with. I decided to go with `rubocop-internal_affairs`, similar to `rubocop-factory_bot`, as I think it is a better name. Additionally, when the legacy setting `rubocop/cop/internal_affairs` is specified, the following warning will be displayed:

```console
Specify `rubocop-internal_affairs` instead of `rubocop/cop/internal_affairs` in your configuration.
```

Such warning is likely to be clearer than documentation :-)